react/hooks/useSelectedFilters.js

```diff
@@ -9,7 +9,7 @@ import { isSameMap } from '../utils/queryAndMapUtils'
  */
 const useSelectedFilters = facets => {
   const { query, map } = useFilterNavigator()
-  if (!query && !map) {
+  if (isNil(query) && isNil(!map)) {
```

Comment:
`isNil(!map)` is always `false`

Reply:
We are also discouraging the overall use of `ramda` in our react apps (check the newest [eslint config here](https://github.com/vtex/typescript/blob/master/packages/eslint-config-vtex-react/io.js#L21)).